### PR TITLE
Add steering elections results blog post

### LINKED ISSUE #4545: [steering blog]: new post to communicate steering election results
Add a blog post under steering to communicate the steering election results.

/assign

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #4545

### CODE REVIEW

File: blog/docs/steering/2021-12-14-steering-elections-results.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+![hero-steering-committee](/blog/images/steering-2021.png)
+
+Hello Knative community,
+
+We are happy to announce that [Carlos Santana](https://twitter.com/csantanapr) and [Lance Ball](https://github.com/lance) are our newly elected [Steering Committee](https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md) members with both serving a 2-year term. Please take a moment to congratulate them!
+
+The competition was fierce with 6 very strong candidates to choose from, so thank you to everyone who ran in this election and everyone who took the time to cast your ballot!
+
+As announced recently as part of [Knative’s application to become a CNCF](https://knative.dev/blog/steering/knative-cncf-donation/) incubating project, [April Nassi](https://twitter.com/thisisnotapril) will remain on the Steering Committee for one more year to help coordinate the transition of infrastructure and ensure a smooth process to bring Knative into the CNCF.
```
Comment (lance) [inline blog/docs/steering/2021-12-14-steering-elections-results.md:14]: Nit
```suggestion
As announced recently in [Knative’s application](https://knative.dev/blog/steering/knative-cncf-donation/) to become a CNCF incubating project, [April Nassi](https://twitter.com/thisisnotapril) will remain on the Steering Committee for one more year to help coordinate the transition of infrastructure and ensure a smooth process to bring Knative into the CNCF.
```

**Review by lance — COMMENTED** [review]
Just a minor wordsmithing / link nit
/approve

File: blog/docs/steering/2021-12-14-steering-elections-results.md (hunk 1 above)
Comment (csantanapr) [reply blog/docs/steering/2021-12-14-steering-elections-results.md:14]: Great @lance thanks for the review

**Comment by csantanapr** [comment]
@lance I made the requested changes

**Review by lance — COMMENTED** [review]
/lgtm

**Pull request merged**